src/Format/Msg/File.h

```diff
@@ -51,10 +51,11 @@ class File : public Dat::Item
     ~File();
 
     std::vector<Message*>* messages();
-    Message* message(unsigned int number);
+    Message* message(unsigned long number);
 
 protected:
-    std::vector<Message*> _messages;
+    std::map<unsigned long, Message*> _messages_map;
+    std::vector<Message*>             _messages_vector;
```

Comment:
Looking at the class declaration, it's not clear why you need 2 containers for messages. Add comments for each new member. And fix style.